.github/ISSUE_TEMPLATE/1-Bug_report.md

```diff
@@ -62,3 +62,13 @@ labels: 'bug'
 - Version or Branch used :
 - Operating System and version :
 - Link to your project :
+
+<!---
```

Comment:
YES!

Reply:
On second thought, we should also add this comment to feature request template as well